api/base/serializers.py

```diff
@@ -13,6 +12,27 @@
 from api.base.exceptions import InvalidQueryStringError, Conflict
 
 
+def format_relationship_links(related_link, self_link, rel_meta, self_meta):
+
+    ret = {
+        'links': {
+            'related': {
+                'href': related_link,
+                'meta': rel_meta
+            },
+            'self': {
+                'href': self_link,
+                'meta': self_meta
+            }
+        }
+    }
+
+    # Not currently returning self links
+    if not ret['links']['self']['href']:
```

Comment:
Though I'm not sure--given our current implementation of `Node.parent_node`, `Node.forked_from`, etc.--we would distinguish between a non-existent vs. null relationship. 

So we are faced two alternatives:

1. Represent `parent` and `forked_from` as `null` for top-level nodes and non-forks?`RelationshipField` just serializes `None` to `None`.
2. Don't include `parent` and `forked_from` in `/data/relationships` for top-level nodes and non-forks. `Node#parent_node` and `Node#forked_from` would probably need to be modified to return a special value in these cases.

Thoughts, @brianjgeiger ?